xcarchive/ios.go

```diff
@@ -378,3 +379,38 @@ func (archive IosArchive) BundleIDProfileInfoMap() map[string]profileutil.Provis
 func (archive IosArchive) FindDSYMs() ([]string, []string, error) {
 	return findDSYMs(archive.Path)
 }
+
+// ReadCodesignParameters ...
```

Comment:
Is there explicit support needed for App Clips?

Reply:
I do not think so, you can not create a valid archive containing an AppClip as the main Product. 
Actually, you can create an AppClip archive but it seems to be invalid (it does not contain any Product or executable file, probably an Xcode bug).

Reply:
Okay, good to know.